### ISSUE TITLE
[Bug] BFR: "Cannot read property 'version' of undefined"

### ISSUE THREAD

**Issue by williamjcm** [rauenzi/BetterDiscordAddons#177]
**Which plugin/theme is this about?**
BetterFormattingRedux

**Describe the Bug**
When launching BBD with BetterFormattingRedux, there's an modal saying "load() could not be fired", with "Cannot read property 'version' of undefined".

**To Reproduce**
Launch BBD with BFR enabled.

**Expected Behavior**
BetterFormattingRedux starts normally

**Screenshots**
- Modal:
![image](https://user-images.githubusercontent.com/13572456/66250614-76004680-e745-11e9-8d44-77d58c302f08.png)
- Errors in console:
![image](https://user-images.githubusercontent.com/13572456/66250630-89131680-e745-11e9-866b-946834602b71.png)

**Discord Version**
Stable

**Additional Context**
Versions used:
- BD: 0.3.2,
- BBD: 0.2.17,
- BFR: 2.3.6,
- ZeresPluginLibrary: 1.2.7

If you need more details, don't hesitate to ask.

**Comment by PhantomGamers** [comment]
I was having this issue as well, deleting `%appdata%/BetterDiscord/plugins/BetterFormattingRedux.config.json` fixed it for me. I looked inside and it was filled with null data, perhaps it's the same for you.

I'm sure it's useless (it's literally just filled with x00s), but I've attached the file anyway.
[BetterFormattingRedux.config.json.txt](https://github.com/rauenzi/BetterDiscordAddons/files/3753654/BetterFormattingRedux.config.json.txt)
